openshift/.env.test

```diff
@@ -1,4 +1,4 @@
-RACK_ENV=preview
+RACK_ENV=staging
```

Comment:
According to above check, `staging` would not be testable, no? I mean the file is `.env.test`..

Reply:
I don't know what is this `.env.test` for but it's not used for launching tests, that's sure, because this file was there before we added support for dotenv. Must be some list of vars loaded when building the image. Since we replaced `preview` by `staging` in all places, the expected behavior should be the same as before.

Reply:
now hopefully it won't start to be used... Will it be automaticlaly loaded now?

Reply:
Because we add something that was not used and then we will be thinking that it is actually needed. If not needed, better get rid of it.

Reply:
I'm not sure whether it's used or not, but I think the changes in this PR doesn't affect this behavior. It only affects container image build.

Reply:
But we also added the dotenv (for whatever unknown reason), and probably now we have files being loaded :man_facepalming:

Reply:
No because it only loads `.env.test` in the source root, it ignores this one.